### PR TITLE
Create DeePMD-kit_Quick_Start_Tutorial_EN.ipynb

### PR DESCRIPTION
**Pull request by Q-Query**
Create DeePMD-kit_Quick_Start_Tutorial_EN.ipynb with Open in Bohrium logo. It has been ensured that the Bohrium Notebook can be run with just one click.

### CODE REVIEW

File: doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb [hunk 1]
```diff
@@ -0,0 +1,1075 @@
...
+   "cell_type": "markdown",
+   "id": "0a4c680a-abcf-40b9-b00e-fa4615aa91b9",
+   "metadata": {},
+   "source": [
+    "DeePMD-kit requires a `json` format file to specify parameters for training. \n",
+    "\n",
+    "In the model section, the parameters of embedding and fitting networks are specified.\n",
+    "```json\n",
+    "\"model\":{\n",
+    "    \"type_map\":    [\"H\", \"C\"],                 # the name of each type of atom\n",
```
Comment (njzjz) [inline doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb:370]: This is not the correct way to comment on a JSON file.

File: doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb [hunk 2]
```diff
@@ -0,0 +1,1075 @@
...
+    "! if ! [ -e colombo-academy-tutorials ];then git clone https://gitee.com/deepmodeling/colombo-academy-tutorials.git;fi;"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f98ce5f8-fb36-4e00-962f-028011801a53",
+   "metadata": {},
+   "source": [
+    "## General Introduction\n",
+    "This tutorial will introduce you to the basic usage of the DeePMD-kit, taking a gas phase methane molecule as an example. [DeePMD-kit's document](https://docs.deepmodeling.org/projects/deepmd/en/master/index.html \n",
```
Comment (njzjz) [inline doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb:107]: In this file, I prefer relative links to absolute links as people read this file inside the documentation.

File: doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb [hunk 3]
```diff
@@ -0,0 +1,1075 @@
...
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "plt.scatter(training_systems[\"energies\"], predict[\"energies\"])\n",
+    "x_range = np.linspace(\n",
+    "    np.min(training_systems[\"energies\"]) * 0.999,\n",
+    "    np.max(training_systems[\"energies\"]) * 1.001,\n",
```
Comment (njzjz) [inline doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb:978]:
```suggestion
    "    *plt.xlim(),\n",
```

File: doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb [hunk 4]
```diff
@@ -0,0 +1,1075 @@
...
+    "```\n",
+    "pair_style  deepmd graph.pb\n",
+    "pair_coeff  * *\n",
+    "```\n",
+    "where the pair style deepmd is invoked and the model file `graph.pb` is provided, which means the atomic interaction will be computed by the DP model that is stored in the file `graph.pb`. \n",
+    "\n",
+    "In an environment with a compatibable version of LAMMPS, the deep potential molecular dynamics can be performed via \n",
+    "\n",
+    "```bash\n",
+    "lmp -i input.lammps\n",
```
Comment (njzjz) [inline doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb:1030]: Is there any reason not to perform it?

**Review by njzjz — CHANGES_REQUESTED** [review]

File: doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb (hunk 1 above)
Comment (Q-Query) [reply doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb:370]: Delete the note in JSON file, and add a table below to explain the parameters.

File: doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb (hunk 4 above)
Comment (Q-Query) [reply doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb:1030]: The `bohrium notebook` image used by this notebook does not have Lammps installed.

Comment (njzjz) [reply doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb:1030]: It is not too hard to install one, using `pip install --force-reinstall deepmd-kit[gpu,cu11,lmp]`

**Comment by Q-Query** [comment]
@Q-Query
[Update quick_start.ipynb](https://github.com/deepmodeling/deepmd-kit/pull/2459/commits/5f234215bcaa27dc61a751fb5f7d737ca23e8235)

**Review by njzjz — APPROVED** [review]

**Review by AnguseZhang — APPROVED** [review]

**Pull request merged**